apps/src/util/i18nStringTrackerWorker.js

```diff
@@ -0,0 +1,104 @@
+import $ from 'jquery';
+
+/**
+ * Gets the singleton instance of an I18nStringTrackerWorker
+ * @returns {I18nStringTrackerWorker}
+ */
+export function getI18nStringTrackerWorker() {
+  return new I18nStringTrackerWorker();
+}
+
+/**
+ * A singleton class which buffers i18n string usage data and sends it in batches to the '/i18n/track_string_usage' API.
+ */
+class I18nStringTrackerWorker {
```

Comment:
is the name "Worker" still accurate?

Reply:
I think it is still accurate. To me, "worker" implies something which does work asynchronously in the background. Although its no longer a web worker, I still like that the name gives context for what it does.